index.Rmd

```diff
@@ -103,15 +103,27 @@ You will be asked to give your credentials (username + password) and then the (a
+The Juno pipelines usually run automated. Minimum input of the user is required. Detailed information on the required input can be found in the section for the specific pipeline. Sometimes a pipeline can ask for input from the user to agree on installing software or a database ([Figure 1.5](#installation-fig) step 1), if you get asked to give permission, please do so in order to run the pipeline. This is most often in the form of pressing y followed by enter ([Figure 1.5](#installation-fig) step 2).
 
-Once the installation is over, the pipeline should start running. Please check regularly that the pipeline is actually running and working. Every time you will get <span style=”color:yellow;”>yellow</span> or <span style=”color:green;”>green</span> messages that tell you what is happening and that indicate that the pipeline is working as expected. Multiple steps are performed simultaneously to optimize the time and resources, especially when using the cluster. If a step fails, you will likely see some <span style=”color:red;”>red</span> text appearing on the screen, but the pipeline might keep going to finish all the steps that are possible. The messages on the screen regularly tell you how far the pipeline is (it gives a % of the steps that have been finished). 
+The first time a Juno pipeline is executed, the preparation might take longer than expected. This is due to the installation of the basic software that is required for the pipeline to run. Be patient! You can recognize that the pipeline is still preparing when it shows a blinking box ([Figure 1.5](#installation-fig) step 3) or when it keeps printing lines inside the terminal.
 
-Once your pipeline is finished you should see some final messages informing you that 100% of the steps were finished and that your output was created. 
+```{r installation-fig, echo = FALSE, eval = TRUE, fig.cap = "Screenshot of the Juno-typing pipeline in the terminal. The pipeline asks to create an environment(step 1). User input is required. To proceed the pipeline press y followed by enter. If executed correctly the pipeline will now show that it is creating the environment(step 2). The blinking box at the bottom indicates that the pipeline is still installing/updating(step 3)."}
+knitr::include_graphics("figures/show_installation.png")
+```
+
+After installation the Juno pipeline will start running. Now the terminal shows <span style="color:#f4d03f;">yellow</span> and/or <span style="color:#00b16a;">green</span> messages. These messages indicate that the pipeline is being executed ([Figure 1.6](#snakemake-complete-run-fig)). At the start of the yellow messages the number of steps is shown ([Figure 1.6](#snakemake-complete-run-fig) step 1).  The amount of steps indicate how much time it will take for the pipeline to finish. Once the pipeline is finished there will be a message printed that all the steps are performed ([Figure 1.6](#snakemake-complete-run-fig) step 2).
```

Comment:
The yellow (with the new hex color) is not well visible. Same for the screenshot... it seems to be very small and with bad resolution. Maybe better to split in two again so it is easy to see the text. Also, something else. the part in which you say that "The amount of steps indicate how much time it will take for the pipeline to finish" is not true. Some steps take way longer than others and the % just tells you how many are missing. Maybe explain that a bit?

Reply:
Fixed